src/SourceBuild/content/eng/Versions.props

```diff
@@ -25,6 +25,8 @@
     <PrivateSourceBuiltSdkVersion>9.0.100-alpha.1.24067.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-alpha.1.24067.1</PrivateSourceBuiltArtifactsVersion>
     <PrivateSourceBuiltPrebuiltsVersion>0.1.0-9.0.100-9</PrivateSourceBuiltPrebuiltsVersion>
+    <PrivateSourceBuiltNoTargetsSdkVersion>3.7.0</PrivateSourceBuiltNoTargetsSdkVersion>
+    <PrivateSourceBuiltTraversalSdkVersion>3.4.0</PrivateSourceBuiltTraversalSdkVersion>
```

Comment:
I think those versions should come from the VMR root's global.json file. Having a single source of truth makes more sense than duplicating the version here. You can use a Regex in msbuild to retrieve the version to avoid writing a custom task.

Reply:
Good idea - will do that.

Reply:
Hmm - if I do this, I won't be able to initialize SDK overrides in `Directory.Builld.props`. And, I need that so that SDK can be extracted. SBRP project references SDK overrides, before any targets are executed:
```
  <ItemGroup>
    <BuiltSdkPackageOverride Include="@(NoTargetsSdkOverride)" />
    <BuiltSdkPackageOverride Include="@(TraversalSdkOverride)" />
```

The only way this would work is to revert to the old code and extract all available SDKs into sub-folders.

Reply:
Why? Regex can be used as a property function in msbuild.

Reply:
I see - and this would work with a file (global.json) as input, and not just with another property?

Reply:
Yes: `$([System.IO.File]::ReadAllText($FilePath))` ->

`<Out>$([System.Text.RegularExpressions.Regex]::Match('$([System.IO.File]::ReadAllText($FilePath))', 'pattern'))</Out>`

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/27b92a3eee2f070b326230688f4dad59299358cd